index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
+      integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="CSS/styles.css" />
+    <link rel="stylesheet" href="CSS/main.css" />
+    <title>Natnael-portfolio</title>
+  </head>
+  <body>
+    <section class="container">
+      <nav class="normal-button">
+        <div class="logo">
+          <div class="placeholder">
+            <div><a href="/"></a></div>
+            <div>N-A</div>
+          </div>
+        </div>
+
+        <button type="button" class="hamburger" id="hamburger">
+          <i class="fas fa-bars"></i>
+        </button>
+        <button type="button" id="close" class="hide">
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="nav-ul">
+          <li><a href="#portfolio" class="nav-text-1">Portfolio</a></li>
+          <li><a href="#about" class="nav-text-2">About</a></li>
+          <li><a href="#contact" class="nav-text-3">Contact</a></li>
+        </ul>
+      </nav>
+
+      <section class="headline">
+        <div class="bg-image-1">
+          <div class="text-container">
+            <h1 class="lg-text-1">I’m Natnael Glad to see you!</h1>
+            <h2 class="sm-text-1">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </h2>
+            <h3 class="lg-text-3 slidein">Let's connect</h3>
+          </div>
+          <ul class="social-media">
+            <li><img src="./image/twitter.svg" alt="icon" /></li>
+            <li><img src="./image/linkedin.svg" alt="icon" /></li>
+            <li><img src="./image/M.svg" alt="icon" /></li>
+            <li><img src="./image/cat.svg" alt="icon" /></li>
+            <li><img src="./image/victory.svg" alt="icon" /></li>
+          </ul>
+        </div>
+      </section>
+
+      <section id="portfolio">
+        <section id="portfolio-1"></section>
+        <section id="portfolio-2"></section>
+        <section id="portfolio-3"></section>
+        <section id="portfolio-4"></section>
+      </section>
+      <section id="myModal" class="modal"></section>
```

Comment:
There's nothing showing on your 'works' section. I only see your Headline, About, and a Contact form. Kindly look into your script and make sure that your data is rendered properly.

- I should see all your portfolio projects that are visible on the Figma template. When I click on the `see project` button on each project it should show the popup modal.

![image](https://user-images.githubusercontent.com/38088818/169637803-49d5cc1c-68a2-4ca2-8406-d6aa71b3ac56.png)